src/relay.cpp

```diff
@@ -638,8 +639,17 @@ void callback(evutil_socket_t fd, short event, void *arg) {
     }
 
     auto udp_header = parse_udp(current_position, &tmp);
+    uint16_t udp_len = ntohs(udp_header->len);
+    if (((current_position + udp_len) != ((uint8_t *)ptr + len)) || (udp_len < 8)) { // UDP header length is 8
```

Comment:
!= is too strict, change to '>'?

Reply:
Do we not expect any potential padding in the end of the UDP packet?

Reply:
Padding is included in the udp header length